cmd/draft/logs.go

```diff
@@ -47,8 +49,8 @@ func newLogsCmd(out io.Writer) *cobra.Command {
 	}
 
 	f := cmd.Flags()
-	f.Int64Var(&lc.logLines, "tail", 100, "lines of recent log lines to display")
-
+	f.BoolVar(&lc.tail, "tail", false, "tail the logs file as it's being written")
+	f.UintVar(&lc.line, "line", 20, "line location to tail from (offset from end of file)")
```

Comment:
out of curiosity, can one set `--line -1` to tail from the very beginning of the file?

Reply:
Nope: `Error: invalid argument "-1" for "--line" flag: strconv.ParseUint: parsing "-1": invalid syntax`